models/quality_measures/intermediate/nqf0034_colorectal_cancer_screening/quality_measures__int_nqf0034__performance_period.sql

```diff
@@ -2,35 +2,94 @@
      enabled = var('quality_measures_enabled',var('claims_enabled',var('clinical_enabled',var('tuva_marts_enabled',False))))
    )
 }}
+/*
+    set performance period end to the end of the current calendar year
+    or use the quality_measures_period_end variable if provided
+*/
+with period_end as (
 
-{%- set performance_period_end = var('quality_measures_period_end') -%}
+    select
+        {% if var('quality_measures_period_end',False) == False -%}
+        {{ last_day(dbt.current_timestamp(), 'year') }}
+        {% else -%}
+        '{{ var('quality_measures_period_end') }}'
+        {%- endif %}
+         as performance_period_end
```

Comment:
Added logic to use the QM var if present; otherwise, defaults to end of current calendar year. Same logic added to the BCS measure.